### ISSUE TITLE
Nachricht an Alexa schicken

### ISSUE THREAD

**Issue by Negalein** [MiRo1310/ioBroker.telegram-menu#16]
Hallo Michael!

Eventuell ist sowas auch noch möglich!

Ich hatte ein Blockly (musste ich deaktivieren, da der Telegram-Menu Adapter reingefunkt hat) mit dem es möglich ist, über Telegram Nachrichten (.speak) an Alexa zu schicken.

Ich schreib in Telegram das Keyword (im Blockly für jeweiligen Alexa konfiguriert), schreib dann den Text und der vorher ausgewählte Alexa plappert dies dann.

![image](https://user-images.githubusercontent.com/45309436/233866656-5877fd44-2ab4-4cb6-8e18-26b47e58b233.png)
![image](https://user-images.githubusercontent.com/45309436/233866671-6cc96f9f-ada5-4fff-86e6-e2f008cea234.png)

**Comment by MiRo1310** [comment]
Das müsste doch schon funktionieren. Du nutzt setstate als value den Text und die id des datenpunkts von Alexa speak. 
oder ich verstehe das total falsch. 🫣

**Comment by Negalein** [comment]
> Das müsste doch schon funktionieren. Du nutzt setstate als value den Text und die id des datenpunkts von Alexa speak. oder ich verstehe das total falsch. 🫣

Ja, aber dann kann ich nur die fest konfigurierten Texte verwenden, die in Value stehen?

So zB plappert sie "Test"
![image](https://user-images.githubusercontent.com/45309436/233914280-f9ebc2d6-2912-4d9c-aaa9-23533b6acf8d.png)

Aber ich würde gern nach drücken des Buttons den Text individuell (in Telegram) eingeben.

**Comment by MiRo1310** [comment]
Also drückst den Button, dann kommt in Telegramm die Abfrage was gesprochen werden soll, du gibst den Text ein und sendest. 
Ich muss jetzt mal fragen wozu man das braucht.

**Comment by Negalein** [comment]
Genau. Mit dem Blockly hab ich zB in Telegram "Kinderzimmer" geschrieben.
Dann den Text der gesprochen werden soll zB "80;Essen kommen" (80 ist hier die Lautstärke).

>Ich muss jetzt mal fragen wozu man das braucht.

darauf hab ich gewartet ;)

Ich hab das immer gerne benutzt, da es schneller als ein Drop in geht und ich die Lautstärke des gesprochenem individuell mitgeben kann.

**Comment by MiRo1310** [comment]
Aber eigentlich könntest du doch etwas im Telegram schreiben und dein blockly darauf reagieren lassen. Mein Adapter würde sagen Befehl nicht gefunden. Aber das blockly würde auslösen

**Comment by MiRo1310** [comment]
Ansonsten melde dich noch mal
